activetable.xml

```diff
@@ -63,10 +64,24 @@ def init(table):
         [cell.get_state() if isinstance(cell, ResponseCell) else str(cell) for cell in row]
         for row in table
     ]
+    initial_state = dict(
+        data=data,
+        help_text=help_text,
+        column_widths=column_widths,
+        row_height=row_height,
+    )
```

Comment:
I intentionally chose the syntax with less punctuation.  I don't like the visual noise of the colons and single quotes, but I'm happy to change it if you prefer.